Gemfile

```diff
@@ -33,19 +33,19 @@ group :development do
   gem "rubocop", '~> 1.50.0',                    require: false
   gem "rubocop-performance", '= 1.16.0',         require: false
   gem "rubocop-rspec", '= 2.19.0',               require: false
-  gem "puppet-strings", '~> 4.0',                require: false
   gem "rb-readline", '= 0.5.5',                  require: false, platforms: [:mswin, :mingw, :x64_mingw]
+  gem "rexml", '>= 3.0.0', '< 3.2.7',            require: false
```

Comment:
There's a [CVE](https://www.ruby-lang.org/en/news/2024/08/22/dos-rexml-cve-2024-43398/) for this version of `rexml`